src/main/java/reposense/system/CommandRunner.java

```diff
@@ -176,6 +182,26 @@ private static String convertToGitDateRangeArgs(Date sinceDate, Date untilDate)
         return gitDateRangeArgs;
     }
 
+    /**
+     * Returns the {@code String} command to specify the authors to analyze for `git log` command.
+     */
+    private static String convertToFilterAuthorArgs(Author author) {
+        StringBuilder filterAuthorArgsBuilder = new StringBuilder(" --author=\"");
+        // ignore check against email
+        final String cmdFormat = "\\|^%s <.*>$";
+
+        filterAuthorArgsBuilder.append(String.format(cmdFormat, author.getGitId()).substring(2));
+
+        // git author names may contain regex meta-characters, so we need to escape those
+        author.getAuthorAliases().stream()
+                .map(authorAlias -> String.format(
+                        cmdFormat, escapeSpecialRegexChars(authorAlias.replace("\\", "\\\\\\"))))
```

Comment:
Perhaps the `.replace("\\", "\\\\\\")` can be included into `escapeSpecialRegexChars` as well

Reply:
If `\\` is also placed into `escapeSpecialRegexChars`, only 1 extra backslash will be added to it, but the `git` author regex pattern requires 2 backslashes, which is why I needed to do a separate escape for it